src/modules/control.cpp

```diff
@@ -34,13 +34,70 @@ static uint16_t targetVolume;
 static uint32_t totalBreathTime;
 static uint32_t targetInhalationTime;
 
+// Added
+static uint32_t targetExhalationTime;
+
+// Added
+static uint16_t targetPressure;
```

Comment:
Are not implementing pressure control at the moment, only requirement modes requested as CMV and SIMV -- its unclear where CMV is supposed to support assisted breathing so the modes are VC == non-assisted CMV, AC == assisted CMV, SIMV allows for spontaneous breaths, though its unclear to me if they need to be assisted

Reply:
My understanding is that AC is pressure controlled. Am I wrong?

Reply:
Ok! Let me know about SIMV requirements as you receive them.